qutebrowser/browser/hints.py

```diff
@@ -748,6 +748,38 @@ def start(self, rapid=False, group=webelem.Group.all, target=Target.normal,
         self._context.tab.find_all_elements(selector, self._start_cb,
                                             only_visible=True)
 
+    def _get_visible_hints(self):
+        """Get elements which are currently visible.
+        """
```

Comment:
Whoa, thanks for this... Thanks to this I just found out how `flake8` doesn't actually run properly anymore :worried: (because one-line docstrings should have the `"""` on the same line, and flake8 should definitely catch that)